### ISSUE TITLE
feature: `kpm pkg` needs to support include/exclude mechanism.

### ISSUE THREAD

**Issue by zong-zhe** [kcl-lang/kpm#306]
## Feature Request

**Is your feature request related to a problem? Please describe:**

`kpm pkg` needs to support include/exclude mechanism during the packaging process to support the selection of files in the package.

**Describe the feature you'd like:**

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

**Comment by officialasishkumar** [comment]
Can you please assign me this issue?

**Comment by officialasishkumar** [comment]
My current proposal for adding this feature is to have include and exclude in `kcl.mod` like this:
```toml
[package]
name = "my_package"
edition = "v0.8.0"
version = "0.0.1"
include = ["/src", "*."]
exclude = [".*", "dist"]
```
I will make a function that will exclude and include the files/paths mentioned in the include/exclude key. 

cc: @Peefy @zong-zhe

**Comment by Peefy** [comment]
Closed by #316
